php/libraries/NDB_Caller.class.inc

```diff
@@ -68,13 +68,11 @@ class NDB_Caller
     * @param string $test_name The object type to load.  The test_name of the
     *                          menu, form or instrument
     * @param string $subtest   The subtest.  The page of the form or instrument.
-    * @param string $CommentID The commentid being loaded.
-    * @param string $nextpage  A page to forward to on completion of save.
-    *                          Used in direct data entry.
+    * @param string $anonymous True if the user has not logged in.
     *
     * @return string rendered object, as html
     */
-    function load($test_name, $subtest, $CommentID = '', $nextpage=null)
+    function load($test_name, $subtest, $anonymous=true)
```

Comment:
Why was comment ID removed from the function signature?

Reply:
I was going to say because it's not passed at the only calling site: https://github.com/aces/Loris/blob/minor/htdocs/main.php#L171

But then I realized that it's used in survey.php too, where the parameters are used, so I'll add it back. Thanks for pointing that out!

Reply:
Fixed